pkg/validation/validate.go

```diff
@@ -0,0 +1,31 @@
+package validation
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+const (
```

Comment:
should these be moved to _common_?

Reply:
i don't think so, they're specific only to this package.  is there a case where they might be used generally?